/thermal_imgproc:
  ros__parameters:
    image_topic: thermal_raw
    camera_name: cam_thermal
    image_width_origin: 1280
    image_height_origin: 512
    show_img: false
    use_sensor_data_qos: false